docs/source/callbacks.rst

```diff
@@ -104,6 +104,16 @@ We successfully extended functionality without polluting our super clean
 
 ----------------
 
+Persiting State
+---------------
+
+If you need to persist your callback's state to be reloaded from a checkpoint,
```

Comment:
I think the beginning and end of this sentence does not match. Not sure actually :sweat_smile:

Reply:
I updated the description, is it more clear now?

Reply:
yes, thank you. 
Also could you check my comment here 
https://github.com/PyTorchLightning/pytorch-lightning/pull/2501#discussion_r471037613
in case you haven't seen it yet :)